src/flamenco/runtime/fd_executor.c

```diff
@@ -387,6 +390,19 @@ fd_executor_load_transaction_accounts( fd_exec_txn_ctx_t * txn_ctx ) {
       acct->starting_lamports = acct->meta->info.lamports;
     }
 
+    /* https://github.com/anza-xyz/agave/blob/v2.0.9/svm/src/account_loader.rs#L222-L277 */
+    int is_sysvar_instruction = !memcmp( acct->pubkey->key, fd_sysvar_instructions_id.key, sizeof(fd_pubkey_t) );
+    int account_exists = fd_acc_exists( acct->const_meta );
+    int is_fee_payer = (i == 0);
```

Comment:
you dont need this (look at the for loop bounds)

Reply:
thanks